ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/mock/MockKafkaTopicClient.java

```diff
@@ -54,6 +57,14 @@ public boolean isTopicExists(final String topic) {
     return Collections.emptyMap();
   }
 
+  @Override
+  public TopicDescription describeTopic(final String topicName) {
```

Comment:
Rather than hard coding your test requirements into the generic `MockKafkaTopicClient` you might be better of switching your test to use `FakeKafkaTopicClient`, which allows you to set up preconditions, like topics.

Reply:
Done!